### PR TITLE
[Pipelines] Doc changes for markdown viewer

### PR DESCRIPTION
**Pull request by yebrahim**
The changes are made here: https://github.com/kubeflow/pipelines/pull/897.

This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/517)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-517--competent-brattain-de2d6d.netlify.com/docs/pipelines/output-viewer/

File: content/docs/pipelines/output-viewer.md [hunk 1]
```diff
@@ -145,3 +145,16 @@ must be self-contained, with no references to other files in the filesystem. It
...
+## Markdown
+
+**type:** `'markdown'`
+
+**Metadata fields:**
+
+- `storage`
+- `source`
+
+Renders markdown strings in the output. The markdown data can either be read from
```
Comment (sarahmaddox) [inline content/docs/pipelines/output-viewer.md]: s/markdown/Markdown

Here and in the next sentence.

File: content/docs/pipelines/output-viewer.md [hunk 2]
```diff
@@ -145,3 +145,16 @@ must be self-contained, with no references to other files in the filesystem. It
...
+a file stored remotely, or can be embedded in the metadata field `source`, in
+which case the `storage` field's value must be `"inline"`.
```
Comment (sarahmaddox) [inline content/docs/pipelines/output-viewer.md]: For consistency with the rest of the doc, use single quotation marks: `'inline'`

**Comment by sarahmaddox** [comment]
/LGTM

I'll wait for @jlewi to approve, since he's also listed as assignee on this PR.

**Comment by jlewi** [comment]
@sarahmaddox Its a bad idea to block on me for PRs :) Don't put too much fail in the auto assignment its not that good. 

/lgtm
/approve

**Pull request merged**